### PR TITLE
COMMERCE-9877 Add account name searchablity to orders

### PR DESCRIPTION
**Pull request by dsitu**
https://issues.liferay.com/browse/COMMERCE-9877

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderLocalServiceImpl.java [hunk 1]
```diff
@@ -659,6 +659,14 @@ public List<CommerceOrder> getCommerceOrders(
 			groupId, commercePaymentMethodKey);
 	}
 
+	@Override
+	public List<CommerceOrder> getCommerceOrdersByAccount(
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderLocalServiceImpl.java:663]: I really like the idea of the ModelListener for AccountEntry name changes!

But this method, isn't it pretty much the same as

public List<CommerceOrder> getCommerceOrdersByCommerceAccountId(
		long commerceAccountId, int start, int end,
		OrderByComparator<CommerceOrder> orderByComparator)

that you can find at line 671(679 with your changes)?

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Review by luca-pellizzon — APPROVED** [review]

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:reevaluate:1347401_2723

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce

**Pull request closed without merging**